core/src/main/scala/org/http4s/AttributeMap.scala

```diff
@@ -85,6 +77,6 @@ object AttributeMap
 // type inference required less generality
 /** A map entry where `key` is constrained to only be associated with a fixed value of type `T`. */
 final case class AttributeEntry[T](key: AttributeKey[T], value: T) {
-  override def toString: String = key.name + ": " + value
+  override def toString: String = s"$key: $value"
```

Comment:
Hmm.  `key` is no longer going to have a useful `toString`.

Does anybody know offhand how sbt's `project` macro works?  It gets the name of the value to which it's assigned.  If it's stupid simple, that might improve debugging.  If it's non-trivial, never mind.